src/components/settings/SettingsLMSTab/ErrorReporting/utils.jsx

```diff
@@ -0,0 +1,25 @@
+import * as timeago from 'timeago.js';
+import { CheckCircle, Error, Sync } from '@edx/paragon/icons';
+
+timeago.register('time-locale');
+
+export function isStrictlyArray(data) {
+  if (!Array.isArray(data) || !data.every((p) => typeof p === 'object' && p !== null)) {
+    return false;
+  }
+  return true;
```

Comment:
nit: Can save a few lines!
```suggestion
  return Array.isArray(data) && data.every((p) => typeof p === 'object' && p !== null);
```